ngraph/frontend/onnx_import/src/op/conv.cpp

```diff
@@ -80,11 +80,28 @@ namespace ngraph
                     std::shared_ptr<ngraph::Node> add_bias(const Output<ngraph::Node>& ng_conv,
                                                            const Output<ngraph::Node>& bias)
                     {
-                        const auto rank_of_conv = ng_conv.get_partial_shape().rank().get_length();
-
-                        return {std::make_shared<default_opset::Add>(
-                            ng_conv,
-                            reshape::reshape_channel_shaped_node_to_nchw(bias, rank_of_conv))};
+                        const auto conv_shape = std::make_shared<default_opset::ShapeOf>(ng_conv);
+                        const auto conv_rank = std::make_shared<default_opset::ShapeOf>(conv_shape);
+
+                        // Prepare tail shape (rank = conv.rank - 2): [1, 1, 1, 1, ... ]
+                        const auto one_const =
+                            default_opset::Constant::create(element::i64, Shape{1}, {1});
+                        const auto two_const =
+                            default_opset::Constant::create(element::i64, Shape{1}, {2});
+                        const auto tail_shape_rank =
+                            std::make_shared<default_opset::Subtract>(conv_rank, two_const);
+                        const auto tail_shape =
+                            std::make_shared<default_opset::Broadcast>(one_const, tail_shape_rank);
+
+                        // Construct new bias shape: [1, C, 1, 1, ... ]
+                        const auto C_dim = std::make_shared<default_opset::ShapeOf>(bias);
+                        const auto bias_shape = std::make_shared<default_opset::Concat>(
+                            OutputVector{one_const, C_dim, tail_shape}, 0);
+
+                        const auto reshaped_bias =
```

Comment:
Why do we need to reshape bias if Eltwise operations supports broadcast?

Reply:
Since data have shape NCHW and Bias have shape C default broadcasting won't work here.
example:
data: 1, 3, 224, 224
bias: 3
since 3 != 224 the default broadcast will throw an error
to make it work we have to reshape bias to shape 1, 3, 1, 1